geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/VisibilityIterator.scala

```diff
@@ -0,0 +1,38 @@
+/**********************************************************************
+ * Copyright (c) 2013-2021 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ **********************************************************************/
+
+package org.locationtech.geomesa.accumulo.iterators
+
+import org.apache.accumulo.core.client.IteratorSetting
+import org.apache.accumulo.core.client.admin.TableOperations
+import org.apache.accumulo.core.iterators.IteratorUtil.IteratorScope
+import org.apache.accumulo.core.iterators.user.ReqVisFilter
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
+import org.opengis.feature.simple.SimpleFeatureType
+
+object VisibilityIterator {
+
+  val Name = "ReqVisFilter"
+
+  def set(tableOps: TableOperations, table: String): Unit =
+    tableOps.attachIterator(table, new IteratorSetting(30, Name, classOf[ReqVisFilter]))
```

Comment:
Does HBase have a similar filter that could be used?  Maybe it is worth a "todo" or ticket to indicate that adding something like this in HBase is not done / is future work?

Reply:
not sure... given that hbase vis are tacked on as coprocessors that might get messy :P

Reply:
Ayup, I know; that's why I was thinking docs or tickets to let people know that HBase support for this is more work.  The docs changes may have covered this.